# Complex test array, with submessages collapsed into columns

std_msgs/Header header

uint16[] x
uint16[] y
uint32[] secs
uint32[] nsecs
bool[] polarity
